spmd/tensor/parallel/style.py

```diff
@@ -39,19 +39,19 @@ def make_input_shard_1d(
 
     Args:
         input (Union[Tensor, DTensor]):
-            This single tensor will be sharded on dimension `dim`
-            over the 1-D :class:`DeviceMesh`.
+            This single tensor will be sharded on dimension ``dim``
+            over the 1-D :class:``DeviceMesh``.
```

Comment:
n00b question: is there any tool to automatic format the docstring?

Reply:
We need to unify how we make the comment. And I think in PyTorch they do have a tool to track that. Good question, we can follow up with Dev Infra team on this.

Reply:
here is the ref for docstring used in PyTorch and looks like there is no tool for that: https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md#docstring-type-formatting. You can build one if you want and that will be very helpful to all PyTorch engineers.